### ISSUE TITLE
Cannot hide pre-releases

### ISSUE THREAD

**Issue by nitrocode** [keilerkonzept/terraform-module-versions#71]
My terraform
```hcl
module "ecs_alb_service_task" {
  source  = "cloudposse/ecs-alb-service-task/aws"
  version = "0.56.0"
}
```

This shows pre-release for some reason
```
✗ terraform-module-versions check -pre-release=false .
| UPDATE? |         NAME         | CONSTRAINT | VERSION | LATEST MATCHING | LATEST |
|---------|----------------------|------------|---------|-----------------|--------|
| (Y)     | ecs_alb_service_task | 0.56.0     | 0.56.0  |                 | 0.58.0 |
```

https://github.com/cloudposse/terraform-aws-ecs-alb-service-task/releases

**Comment by sgreben** [comment]
do you mean `0.58.0`?  🤔 a pre-release version would be something like `0.58.0-rc1` (per semver spec, https://semver.org/#spec-item-9)

**Comment by sgreben** [comment]
ah, i see now, it says "unstable pre release" in the release title. it should use a standard pre-release tag (`0.58.0-something`) in this case.

**Comment by sgreben** [comment]
That said, I wouldn't be opposed to adding a GitHub API check for this (opt-in, since UX-wise it's somewhat suprising to have this special thing for GH). I think GitHub was wrong to introduce a pre-release flag that's independent of tag semver semantics, but that's just something we have to live with.

**Comment by nitrocode** [comment]
Yep, I meant if a github release is marked as a `pre-release` versus the semver equivalent.
